src/components/Navbar.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { User } from 'react-feather';
+
+const Navbar = () => (
+  <nav className="navbar navbar-expand-lg navbar-light bg-light">
+    <div className="container ">
+      <h1 className="text-primary">Bookstore CMS</h1>
+      <div className="collapse navbar-collapse">
```

Comment:
- [ ] please kindly align the update progress button to the left of that section like in the original design
- [ ] kindly note that the cards and the nav bar have a white background color, and the general page has a gray color instead.. Please, kindly fix that. Keeping an eye to details is very important.
your design
![Screenshot_7](https://github.com/axcell-0/bookstore/assets/65298048/1cf29b97-2c6d-4d1d-9738-2761255328a4)
original
![Screenshot_6](https://github.com/axcell-0/bookstore/assets/65298048/93ecde2a-eb86-4906-a0fc-afb690b259ce)